posts/2020-09-30-microprofile-language-support-liberty.adoc

```diff
@@ -4,52 +4,52 @@ title: "MicroProfile Language Support for Open Liberty Applications"
 author_github: https://github.com/kathrynkodama
-seo-title: MicroProfile Language Support for Open Liberty Applications - OpenLiberty.io
+seo-title: MicroProfile Language Support for Open Liberty applications - OpenLiberty.io
 seo-description: Leverage the new MicroProfile Language Server along with Visual Studio Code tools to develop your cloud-native Java applications with Eclipse MicroProfile APIs and Open Liberty.
-blog_description: "Leverage the new MicroProfile Language Server along with VS Code tools to develop your Eclipse MicroProfile applications with Open Liberty."
+blog_description: "Leverage the new MicroProfile Language Server along with VS Code tools to develop your MicroProfile applications with Open Liberty."
 ---
-= MicroProfile Language Support for Open Liberty Applications
+= MicroProfile Language Support for Open Liberty applications
 Kathryn Kodama <https://github.com/kathrynkodama>
 
-The new MicroProfile Language Server provides added support for developing your Eclipse MicroProfile applications with Open Liberty. Earlier this month, the first release of the Language Server for Eclipse MicroProfile (https://github.com/eclipse/lsp4mp[Eclipse LSP4MP]), which is an open incubator project under the Eclipse Foundation, was published. The Tools for MicroProfile VS Code extension consumes this language server, providing feature-rich support for MicroProfile APIs and properties in VS Code.
+The new MicroProfile Language Server provides added support for developing your MicroProfile applications with Open Liberty. Earlier this month, the first release of the Language Server for Eclipse MicroProfile (https://github.com/eclipse/lsp4mp[Eclipse LSP4MP]), which is an open incubator project under the Eclipse Foundation, was published. The Tools for MicroProfile Visual Studio Code (VS Code) extension uses this language server, providing feature-rich support for MicroProfile APIs and properties in VS Code.
 
-In this blog, we will showcase how the new MicroProfile Language Server, along with existing tooling for VS Code, makes it even easier to develop your Open Liberty applications.
+In this blog, I'll showcase how the new MicroProfile Language Server, along with existing tools for VS Code, makes it even easier to develop your Open Liberty applications.
 
 == Installing the extensions
 
-Install the https://marketplace.visualstudio.com/items?itemName=MicroProfile-Community.vscode-microprofile-pack[MicroProfile extension pack] from the VS Code Marketplace to support developing your Open Liberty applications with MicroProfile. We will use the following extensions from the MicroProfile extension pack:
+Install the https://marketplace.visualstudio.com/items?itemName=MicroProfile-Community.vscode-microprofile-pack[MicroProfile Extension Pack] from the VS Code Marketplace to support development of your Open Liberty applications with MicroProfile. In this blog, we'll use the following extensions from the MicroProfile Extension Pack:
```

Comment:
Should this be `In this blog, I'll use the following` as per the change in the paragraph above?

Or maybe line 16 could be: `In this blog, you will see how the new MicroProfile Language Server, along with existing tools for VS Code...` if we are aiming for the reader to follow along with the blog